proto/tripcommon.proto

```diff
@@ -145,3 +145,12 @@ enum RoadClass {
   kResidential = 6;
   kServiceOther = 7;
 }
+
+message TaggedName {
```

Comment:
should we add attribute(s) to the `StreetName` instead of creating a new `TaggedName`